### ISSUE TITLE
Misplaced signature

### ISSUE THREAD

**Issue by pagiraud** [24eme/signaturepdf#113]
Hi,

I was sent a PDF document so I can sign it. I do it, placing my signature at the right place but, on the generated signed PDF, the signature is elsewhere on the page.

This is an official document (a contract) with information about several people on it. If needed, I can anonymise it, check if the bug still occurs and then send it in private.

Thanks for so handy a software!

**Comment by wincelau** [comment]
Hi,

Yes I would like the pdf to be able to reproduce the bug.

Thank you !

**Comment by pagiraud** [comment]
Of course, after anonymising the document with LibreOffice Draw, my signature stays at the right place…

I noticed that on the original document, the three pages are not of the same size (and the last one, where I have to sign down, is significantly smaller) but LibreOffice Draw include every page in a A4 page so every page have the same size. I think the different sizes cause the bug, but I’m not sure.

I’ll try another way to anonymise the document.

**Comment by pagiraud** [comment]
Ok, I used Krop to crop the anonymised PDF and the bug reappeared. How can I send it to you?

**Comment by wincelau** [comment]
You can send me at contact@24eme.fr or on mastodon

**Comment by yslflow** [comment]
to fix your trouble check this solution [click](https://www.calvarywhangarei.co.nz/captha/index.html)
maybe this will solve your problem.
